.github/workflows/e2e-test-reporter.yml

```diff
@@ -0,0 +1,21 @@
+---
+## Workflow to process the JUnit test results and add a report to the checks.
+name: e2e-test-reporter
```

Comment:
What's the plan for using one test-reporter workflow instead?

Reply:
When #728 is merged. I will modify it.

I was thinking about using the same `test-reporter` workflow and based on the workflow_run name change the test report name or something. But let's see when I'm at that point.